Week5/build.gradle

```diff
@@ -1,6 +1,11 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 buildscript {
-    ext.kotlin_version = "1.3.72"
+    ext {
+        kotlin_version = "1.3.72"
+        roomVersion = '1.0.0'
+        archLifecycleVersion = '1.1.0'
```

Comment:
dependencies should be declared in app-level build.gradle file, not this one. This is project-level file